pkg/proxy/roundrobin.go

```diff
@@ -227,7 +228,8 @@ func (lb *LoadBalancerRR) OnUpdate(allEndpoints []api.Endpoints) {
 
 	// Update endpoints for services.
 	for _, svcEndpoints := range allEndpoints {
-		key := balancerKey(svcEndpoints.Name)
+		name := types.NewNamespacedName(svcEndpoints.Namespace, svcEndpoints.Name)
+		key := balancerKey(*name)
```

Comment:
you can probably just replace balancerKey with NamespacedName - anything I am missing for that?